app/frontend/src/store/modules/form.js

```diff
@@ -125,6 +126,7 @@ export default {
     isRTL: (state) => state.isRTL,
     subscriptionData: (state) => state.subscriptionData,
     totalSubmissions: (state) => state.totalSubmissions,
+    selectedSubmissions: (state) => state.selectedSubmissions,
```

Comment:
I think we need to start improving the code coverage of the unit tests. Can you check the coverage reports in the actions (or run the tests locally and check app/frontend/coverage/lcov-report/index.html and app/coverage/lcov-report/index.html) to make sure there are tests for anything that is added?

(Although a lot of the tests are fairly simple and don't add a lot of value, it'll make it easier to identify missing coverage if we follow what the original team did for tests)